### PR TITLE
[11.0][MIG] product_supplierinfo_for_customer_invoice

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Standard migration.

### CODE REVIEW

File: product_supplierinfo_for_customer_invoice/models/account_invoice.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+class AccountInvoiceLine(models.Model):
+    _inherit = 'account.invoice.line'
+
+    @api.multi
+    def _get_product_customer_code(self):
+        product_supplierinfo_obj = self.env['product.supplierinfo']
+        for line in self.filtered(lambda il: il.product_id.supplier_ids):
+            product = line.product_id
+            code_id = product_supplierinfo_obj.search([
```
Comment (AaronHForgeFlow) [inline product_supplierinfo_for_customer_invoice/models/account_invoice.py:16]: Does not look good to me. I created a customer invoice and the customer code in the invoice was totally incorrect.

It should filter by the customer, and also the product.supplierinfo must be type customer

**Review by AaronHForgeFlow — CHANGES_REQUESTED** [review]

File: product_supplierinfo_for_customer_invoice/models/account_invoice.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+                ('type', '=', 'customer'),
```
Comment (AaronHForgeFlow) [inline product_supplierinfo_for_customer_invoice/models/account_invoice.py]: `supplierinfo_type`

**Review by AaronHForgeFlow — CHANGES_REQUESTED** [review]

**Review by AaronHForgeFlow — APPROVED** [review]
LGTM now

merge_small

**Review by nikul-serpentcs — APPROVED** [review]
Perfecto :+1:

**Pull request merged**